Configuration/Geometry/python/dict2021Geometry.py

```diff
@@ -2472,12 +2472,12 @@
 allDicts = [ commonDict, trackerDict, caloDict, muonDict, forwardDict, ppsDict ]
 
 detectorVersionDict = {
-    ("O3","T3","C1","M1","F1","P4") : "2021",
-    ("O4","T4","C1","M1","F1","P4") : "2021ZeroMaterial",
-    ("O5","T5","C1","M1","F1","P4") : "2021FlatMinus05Percent",
-    ("O5","T6","C1","M1","F1","P4") : "2021FlatMinus10Percent",
-    ("O5","T7","C1","M1","F1","P4") : "2021FlatPlus05Percent",
-    ("O5","T8","C1","M1","F1","P4") : "2021FlatPlus10Percent",
+    ("O3","T3","C1","M1","F1","P5") : "2021",
+    ("O4","T4","C1","M1","F1","P5") : "2021ZeroMaterial",
+    ("O5","T5","C1","M1","F1","P5") : "2021FlatMinus05Percent",
+    ("O5","T6","C1","M1","F1","P5") : "2021FlatMinus10Percent",
+    ("O5","T7","C1","M1","F1","P5") : "2021FlatPlus05Percent",
+    ("O5","T8","C1","M1","F1","P5") : "2021FlatPlus10Percent",
  }
 
 deprecatedSubdets = set([ "T1", "T2", "O1", "O2", "P1", "P2", "P3" ])
```

Comment:
Hi @bsunanda,

Don't you need to include P4 in deprecatedSubdets because it is not used elsewhere? 

In addition, I still see ppsDict includes P1, P2, P3, 
https://github.com/cms-sw/cmssw/blob/3e7e962e76f3f082aedbe970dc103e68f0976183/Configuration/Geometry/python/dict2021Geometry.py#L1951-L2259

do we need to keep them? For 2026, I think we clean the deprecated ones.

Thanks.

Reply:
Let us keep them for a while till we finalize 2021 - once and for all